packages/react-native-sdk/docusaurus/docs/reactnative/03-core/02-joining-creating-calls.mdx

```diff
@@ -179,3 +179,30 @@ await call.update({
   },
 });
 ```
+
+### Backstage setup
+
+The backstage feature makes it easy to build a use-case where you and your co-hosts can set up your camera before going live.
+Only after you call `call.goLive()` the regular users will be allowed to join the livestream.
+
+However, you can also specify a `join_ahead_time_seconds`,
+which will allow regular users to join the livestream before the call is live, in the specified join time before the stream starts.
+
+Here's an example of how to do that:
+
+```typescript
+await call.getOrCreate({
+  data: {
+    starts_at: new Date(Date.now() + 5000), // 500 seconds from now
```

Comment:
I think this needs to be either `// 5 seconds from now` in the comment or:

```suggestion
    starts_at: new Date(Date.now() + 500000), // 500 seconds from now
```

Reply:
thanks for catching this. I'll fix it right away.